sdcm/localhost.py

```diff
@@ -0,0 +1,41 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+import os
+import logging
+from typing import Optional
+
+from sdcm.utils.docker import ContainerManager
+from sdcm.utils.rsyslog import RSYSLOG_PORT, RSyslogContainerMixin, generate_rsyslog_conf_file
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LocalHost(RSyslogContainerMixin):
+    def __init__(self, user_prefix: Optional[str] = None, test_id: Optional[str] = None) -> None:
```

Comment:
name of this class is a bit vague, it would help to have a few line of docstrings to explain it's purpose
I still didn't fully wrapped my head around what it does

Reply:
There is some resources which are not associated with any of nodes we deploy in clouds. So, it's pretty natural to have special node for such resources.

Reply:
> > the monitoring data should be extract on my home folder, it should go into the logs dir of the monitor.
> > at least it was like that before.
> 
> For Docker backend it uses host as a monitor node. I think this behavior was introduced by Dmitry, but I'm not sure.

I have introduced this code, and seems like it's now missing.